src/shogun/lib/auto_initialiser.h

```diff
@@ -124,14 +124,14 @@ namespace shogun
 								++idx;
 							}
 						}
-						width = GaussianKernel::to_log_width(linalg::median(result));
+						width = linalg::median(result);
```

Comment:
btw is this correct?, I always though that the median would be for a kernel of the form exp(-d² / 2*sigma**2)

Reply:
I will have to read the paper again to confirm, but I think you are right

Reply:
seems to be `width = sqrt(H_n / 2)`, where `H_n` is the median of the pairwise euclidean distance

Reply:
and  then they define in the paper `kernel = f(x,y) / width`, so here f(x,y) = exp(-x**2) and from what I can tell in shogun we compute kernel=exp(-x**2)/width. Hence this line should be `width = std::sqrt(linalg::median(result) / 2)`, I think?

Reply:
ah yes so like you said `width = 2*sigma**2`, where sigma is the median